connectors/katalog/main.go

```diff
@@ -44,6 +48,10 @@ func RunCmd() *cobra.Command {
 			if err != nil {
 				return errors.Wrap(err, "unable to add katalog v1alpha1 to schema")
 			}
+			err = corev1.AddToScheme(scheme)
```

Comment:
why do we need it?

Reply:
it is needed to read secrets. Please see GetCertificatesFromSecret function in pkg/tls:

```func GetCertificatesFromSecret(client kclient.Client, secretName, secretNamespace string) (map[string][]byte, error) {
	secret := &corev1.Secret{
		ObjectMeta: metav1.ObjectMeta{
			Namespace: secretNamespace,
			Name:      secretName,
		},
	}